### ISSUE TITLE
`.sw` dir content in the wrong place

### ISSUE THREAD

**Issue by pfecchio** [alidock/alidock#63]
I have build the AliPhysics and AliRoot software as described in https://alice-doc.github.io/alice-analysis-tutorial/building/alidock.html. After the build in the `home/alidock/.sw` directory I can't find the `BUILD`, `INSTALLROOT`, `MODULES`, etc. dirs. There is only the `MIRROR` dir. 
Looking around I found that the aforementioned directories are actually located in the `/persist/sw` directory that is not directly accessible from outside the container.
I have already shown this unexpected behavior to @mconcas which suggest me to open this issue.

**Comment by mconcas** [comment]
Cheers @pfecchio, 
Finally, I figured out what is the real problem.

Indeed, this is related to #39, which is a MacOS-only issue. 
This is due to the fact we run a full-fledged virtual machines around Docker, and machinery (volumes included) need to run on a Linux host. That's why on native Linux you will not see the following scenario.
 
The content of the canonical `.sw` directory has been moved to a  [docker volume](https://docs.docker.com/storage/) and mounted to the `/persist/sw` path of each container you run with alidock. The `MIRROR` directory is sym-linked from .sw -> /persist/sw, to have sources shared inside/outside alidock and, simultaneously,  not to be penalized by the [HyperKit](https://github.com/moby/hyperkit) overhead in compilation+I/O operations.

Unfortunately it is not something wrong/unexpected and is not easy to change, w/o paying a **lot** in terms of performances. At the moment I don't have an easy solution to make the VS Code IntelliSense work.
